src/NewField.cpp

```diff
@@ -277,20 +283,28 @@ void NewFieldColumn::take_over_mods(NewFieldColumn& other)
 #undef CPY
 }
 
-void NewFieldColumn::set_defective_mode()
+void NewFieldColumn::set_defective_mode(bool mode)
```

Comment:
An enumeration with only on and off values doesn't make sense, and there will never be a third mode.